specs/cli/config-migrate.md

```diff
@@ -6,10 +6,10 @@ In the case where conflicting API client names would be migrated, the command wi
 
 ## Parameters
 
-| Parameters | Required | Example | Description |
-| -- | -- | -- | -- |
-| `--lock-location \| --ll` | No | ./output/pythonClient/kiota-lock.json | Location of the `kiota-lock.json` file. If not specified, all `kiota-lock.json` files within in the current directory tree will be used. |
-| `--client-name \| --cn` | No | GraphClient | Used with `--lock-location`, it would allow to specify a name for the API client. Else, name is auto-generated as a concatenation of the `language` and `clientClassName`. |
+| Parameters | Required | Example | Description | Telemetry | 
+| -- | -- | -- | -- | -- |
+| `--lock-location \| --ll` | No | ./output/pythonClient/kiota-lock.json | Location of the `kiota-lock.json` file. If not specified, all `kiota-lock.json` files within in the current directory tree will be used. | Yes, without its value |
```

Comment:
--ll is already used for the log level parameter, we need to define a different alias

Reply:
No alias then!